shared/src/userData.ts

```diff
@@ -73,6 +74,7 @@ export const createEmptyBusiness = (id?: string): Business => {
     },
     environmentData: undefined,
     version: CURRENT_VERSION,
+    userId: user?.id || createEmptyUser().id,
```

Comment:
[boulder] We invoke `createEmptyBusiness` in `addAdditionalBusiness.ts` and are not passing a user there. Every time an Ethan adds a business they will be creating the business with a random user id.

To safeguard against us doing this in the future, we should require `user` as a param here.

Reply:
It might be good to add a test in `addAdditionalBusiness.test.ts` that validates that all additional businesses have the same user id as the associated user.

I'm ok with that being added as a fast-follow given we're trying to get this out as part of the release.